pkgs/build-support/dotnet/build-dotnet-module/hooks/dotnet-check-hook.sh

```diff
@@ -9,22 +9,27 @@ dotnetCheckHook() {
         local dotnetProjectFilesArray=( "${dotnetProjectFiles[@]}" )
         local dotnetTestProjectFilesArray=( "${dotnetTestProjectFiles[@]}" )
         local dotnetTestFlagsArray=( "${dotnetTestFlags[@]}" )
+        local dotnetTestFiltersArray=( "${dotnetTestFilters[@]}" )
         local dotnetDisabledTestsArray=( "${dotnetDisabledTests[@]}" )
         local dotnetRuntimeDepsArray=( "${dotnetRuntimeDeps[@]}" )
         local dotnetRuntimeIdsArray=( "${dotnetRuntimeIds[@]}" )
     else
         local dotnetProjectFilesArray=($dotnetProjectFiles)
         local dotnetTestProjectFilesArray=($dotnetTestProjectFiles)
         local dotnetTestFlagsArray=($dotnetTestFlags)
+        local dotnetTestFiltersArray=($dotnetTestFilters)
         local dotnetDisabledTestsArray=($dotnetDisabledTests)
         local dotnetRuntimeDepsArray=($dotnetRuntimeDeps)
         local dotnetRuntimeIdsArray=($dotnetRuntimeIds)
     fi
 
     if (( ${#dotnetDisabledTestsArray[@]} > 0 )); then
-        local disabledTestsFilters=("${dotnetDisabledTestsArray[@]/#/FullyQualifiedName!=}")
+        dotnetTestFiltersArray=( "${dotnetTestFiltersArray[@]}" "${dotnetDisabledTestsArray[@]/#/FullyQualifiedName!=}" )
+    fi
+
+    if (( ${#dotnetTestFiltersArray[@]} > 0 )); then
         local OLDIFS="$IFS" IFS='&'
-        dotnetTestFlagsArray+=("--filter:${disabledTestsFilters[*]//,/%2C}")
+        dotnetTestFlagsArray+=("--filter:${dotnetTestFiltersArray[*]//,/%2C}")
```

Comment:
@corngood should `s/,/%2C/` happen here for all filters, or only be done for the `disabledTests`?

Reply:
Actually maybe it should be only for `disabledTests`, since those are test names, and `testFilters` is filter expressions, which should be escaped properly.